autoconfigure-storage/src/main/java/com/google/cloud/trace/zipkin/autoconfigure/ZipkinStackdriverStorageAutoConfiguration.java

```diff
@@ -106,15 +101,22 @@ String getDefaultProjectId() throws IOException {
     HttpURLConnection connection = (HttpURLConnection) url.openConnection();
     connection.setRequestProperty("Metadata-Flavor", "Google");
     connection.setRequestMethod("GET");
-    try (InputStream responseStream = connection.getInputStream())
-    {
+    try (InputStream responseStream = connection.getInputStream()) {
```

Comment:
fyi your formatting changes look legit to me... although these changes move it to google style from what is some variation of that or square.. later it will switch to square :P